client/knowledge_base_search/src/containers/SearchContainer/styles.css

```diff
@@ -1,11 +1,88 @@
 .header--input {
   color: #ffffff;
   text-align: center;
+  margin-bottom: 2rem;
+}
+
+.clickable-tabs {
+  display: table;
+  border-spacing: 1rem;
+  width: 70rem;
+  position: relative;
+  bottom: 3rem;
+}
+
+.clickable-tab {
+  color: #ffffff;
+  border-style: solid;
```

Comment:
you can define all of these border-* in one line: `border: medium solid #f0f0f0`, although not used to using `medium` for a width